modelforge/potential/__init__.py

```diff
@@ -2,29 +2,32 @@
 This module contains the implemented neural network potentials and their parameters.
 """
 
-from .schnet import SchNet
-from .physnet import PhysNet
-from .painn import PaiNN
-from .ani import ANI2x
-from .sake import SAKE
-from .tensornet import TensorNet
+from enum import Enum
+
+from .ani import ANI2xCore
+from .aimnet2 import AimNet2Core
```

Comment:
very minor, but can we group these imports with the other potential core inputs?